flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/sinks/TableSinkUtils.scala

```diff
@@ -61,10 +61,10 @@ object TableSinkUtils {
 
       // format table and table sink schema strings
       val srcSchema = srcFieldNames.zip(srcFieldTypes)
-        .map { case (n, t) => s"$n: ${t.getConversionClass.getSimpleName}" }
+        .map { case (n, t) => s"$n: ${fromDataTypeToLogicalType(t).toString}" }
```

Comment:
Can you create a `srcLogicalTypes` and `sinkLogicalTypes`?
With these two field, should be easier to debug and reduce code duplicate.

Reply:
@JingsongLi yeah, that's better.